interop/test_utils.go

```diff
@@ -655,7 +655,8 @@ func DoPickFirstUnary(tc testgrpc.TestServiceClient) {
 		Payload:      pl,
 		FillServerId: true,
 	}
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	// TODO(mohanli): Revert the timeout back to 10s once TD migrates to xdstp.
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
Given that the watch timeout is `15s`, this leaves only `5s` for the test.  I would request that this be set to `30s` instead.

Reply:
Done.